src/web/routes/application/national-insurance-number/national-insurance-number.js

```diff
@@ -0,0 +1,29 @@
+const { sanitize } = require('./sanitize')
+const { validate } = require('./validate')
+
+const pageContent = ({ translate }) => ({
+  title: translate('nationalInsuranceNumber.title'),
+  heading: translate('nationalInsuranceNumber.heading'),
+  ninoLabel: translate('nationalInsuranceNumber.ninoLabel'),
+  hint: translate('nationalInsuranceNumber.hint'),
+  explanation: translate('nationalInsuranceNumber.explanation'),
+  buttonText: translate('buttons:continue')
+})
+
+const contentSummary = (req) => ({
+  key: req.t('nationalInsuranceNumber.summaryKey'),
+  value: req.session.claim.nino
```

Comment:
This still being `nino` looks like the proverbial sore thumb here...

Reply:
Yes - I discussed with team and decided to leave the claim property as `nino`, but I don't mind renaming this as well - although would this have implications on the claimant service?